frontend/src/pages/Policy/policySections/DataCollectionTab.tsx

```diff
@@ -7,6 +12,194 @@ export default function DataCollectionTab() {
+						System)
+					</a>{' '}
+					dataset includes a wide range of information related to gun-related
+					injuries, providing a holistic perspective on the impact of
+					gun-related incidents.
+				</p>
+				<ul className='list-none grid gap-4 grid-cols-2 p-0 text-smallest'>
+					<li className='flex flex-col'>
+						<p className='my-0 text-altGreen font-semibold'>
+							Time-Series Range
+						</p>
+						<p className='my-0'>2001 - current</p>
+					</li>
+					<li className='flex flex-col'>
+						<p className='my-0 text-altGreen font-semibold'>Geographic Level</p>
+						<p className='my-0'>National, State</p>
+					</li>
+					<li className='flex flex-col'>
+						<p className='my-0 text-altGreen font-semibold'>
+							Demographic Granularity
+						</p>
+						<p className='my-0'>Race/ethnicity, sex, age</p>
+					</li>
+					<li className='flex flex-col'>
+						<p className='my-0 text-altGreen font-semibold'>Update Frequency</p>
+						<p className='my-0'>Yearly</p>
+					</li>
+				</ul>
+				</article>
+			</section>
+			<section id='#key-metrics'>
+				<div className='mb-0'>
+					<p className='my-0 text-left font-sansTitle text-smallest font-extrabold uppercase text-black tracking-widest'>
+						OUR METHODS
+					</p>
+					<Tooltip
+						placement='bottom-start'
+						title={
+							<Typography sx={{ fontSize: '14px', fontWeight: '600' }}>
+								Metrics are quantifiable indicators used to measure and analyze
+								various aspects of public health data.
+							</Typography>
+						}
+					>
+						<h4 className='my-0 text-title font-medium text-altGreen underline decoration-dotted inline'>
+							Key Metrics
+						</h4>
+					</Tooltip>
+				</div>
+				<p>
+					Our key metrics encompass data on fatal gun-related incidents, with a
+					detailed breakdown by age, sex, and, for fatalities, by race and
+					ethnicity (as the data allows). These metrics provide a granular view
+					of the gun violence landscape, highlighting trends and disparities
+					crucial for understanding the impact within the Atlanta community.
+				</p>
+			</section>
+			<section id='#data-limitations'>
+				<h3 className='my-0 text-title font-medium text-altGreen'>
+					Data Limitations
+				</h3>
+				<p>
+					While our dataset is comprehensive, it's important to note certain
+					limitations: potential underreporting in certain demographics, and the
+					challenges in interpreting data with suppressed or unstable values.
+					These limitations necessitate cautious interpretation and underscore
+					the need for ongoing data enhancement.
+				</p>
+			</section>
+
+			<section id='#available-data'>
+				<h3 className='my-0 text-title font-medium text-altGreen'>
+					Available Data
+				</h3>
+				<div className='grid grid-cols-1 md:grid-cols-2'>
+					<div>
+						<p className='text-smallest'>
+							What’s included in our <HetTerm>youth (0-17)</HetTerm> dataset:
```

Comment:
"Youth" for us comprises both "Children 0-17" and "Young Adults 18-25"; I think for this since they have the same breakdowns we can just change this to read 0-25